lib/benchee/system.ex

```diff
@@ -125,7 +125,10 @@ defmodule Benchee.System do
                   |> String.trim()
                   |> String.trim_trailing(" kB")
                   |> Integer.parse
-    {memory_in_bytes, _} = Memory.convert({memory_in_kilobytes, :kilobyte}, :byte) #memory_in_kilobytes * 1024
+
+    {memory_in_bytes, _} = {memory_in_kilobytes, :kilobyte}
+                  |> Memory.convert(:byte)
```

Comment:
using pipe is generally discouraged if there is only one pipe, I think the code can stay as above :)

Reply:
Also I gues this breaks right now because the `convert` code only deals with "real" units as in unit structs as of now. It might be nice to change that (you don't have to here) - but if the supplied unit is we can just call something like `unit_for` with the atom and then call ourselves with the right struct. Might just wrap it in that either way

Reply:
ah ok we already have a special case like that forget the second comment :)

Reply:
I removed the pipe and just put the argument list on separate lines. Originally I had used the pipe to get around a credo error for line length heh.